packages/cli-repl/package.json

```diff
@@ -21,6 +21,7 @@
     "compile": "tsc -p tsconfig.json",
     "start": "node bin/mongosh.js",
     "pretest": "npm run compile",
+    "my-test": "cross-env TS_NODE_PROJECT=../../configs/tsconfig-mongosh/tsconfig.test.json mocha -r \"../../scripts/import-expansions.js\" --timeout 60000 -r ts-node/register \"./{src,test}/**/e2e-fle.spec.ts\"",
```

Comment:
```suggestion
```

(Unless I'm missing something? :))

Reply:
~Oh just missed it, thought I removed it before committing~. You spotted it, it was on purpose 😀. Removing it.